src/main.cpp

```diff
@@ -1,5 +1,5 @@
 // mgconsole - console client for Memgraph database
-// Copyright (C) 2016-2020 Memgraph Ltd. [https://memgraph.com]
+// Copyright (C) 2016-2021 Memgraph Ltd. [https://memgraph.com]
```

Comment:
Just to be 100% sure, all the other changes in this file are from `clang-format` correct?

Reply:
That's correct